examples/axios.test.js

```diff
@@ -883,4 +883,19 @@ describe('axios', function() {
       console.log(res3);
     }));
   });
+  it('axios-dataA', async function() {
+    await axios.get('https://httpbin.org/get').then((res) => {
+      res.data; // Returns the HTTP response body at the server we requested.
+      typeof res.data; // object
+    });
+  });
+  it('axios-dataB', async function() {
+    await axios.get('https://httpbin.org/get', {
```

Comment:
Instead of `await axios.get(...).then(res => {})` you should just do `const res = await axios.get(...);`